packages/plugins/README.md

```diff
@@ -147,12 +147,12 @@ registerPlugin( 'plugin-name', {
 
 _Parameters_
 
--   _name_ `string`: A string identifying the plugin.Must be unique across all registered plugins.
--   _settings_ `Omit<WPPlugin, 'name'>`: The settings for this plugin.
+-   _name_ `string`: A string identifying the plugin. Must be unique across all registered plugins.
+-   _settings_ `PluginSettings`: The settings for this plugin.
 
 _Returns_
 
--   `WPPlugin`: The final plugin settings object.
+-   The final plugin settings object.
```

Comment:
we've lost this return type here, I'm guessing because we lost the return type annotation in the function declaration. in any case, it seems like the existing JSDoc type was wrong since it's `WPPlugin | null` according to what I see.